osf/management/commands/populate_custom_taxonomies.py

```diff
@@ -19,13 +19,14 @@ def validate_input(custom_provider, data, copy=False, add_missing=False):
     # This function may be run outside of this command (e.g. in the admin app) so we
     # need to make sure that BEPRESS_PROVIDER is set
     global BEPRESS_PROVIDER
-    BEPRESS_PROVIDER = PreprintProvider.objects.filter(_id='osf').first()
+    BEPRESS_PROVIDER = AbstractProvider.objects.filter(_id='osf').first()
```

Comment:
Should also specify `type='osf.preprintprovider'`, as `_id`s are no longer guaranteed unique across types